src/AzureExtension/QuickStartPlayground/AzureOpenAIService.cs

```diff
@@ -127,24 +142,43 @@ public async Task<string> GetAICompletionAsync(string systemInstructions, string
         var openAIClient = OpenAIClient;
         ArgumentNullException.ThrowIfNull(openAIClient);
 
+        var prompts = systemInstructions + "\n\n" + userMessage;

```

Comment:
a use message?  Is this scanned in any way to prevent malevolent actors?

Reply:
Yup! See the instructions in `GetRecommendedLanguageAsync` in `Completions.cs`.